.gitlab/pipeline/publish.yml

```diff
@@ -60,9 +57,81 @@ publish-rustdoc:
     - git commit -m "___Updated docs for ${CI_COMMIT_REF_NAME}___" ||
       echo "___Nothing to commit___"
     - git push origin gh-pages --force
+    # artificial sleep to publish gh-pages
+    - sleep 300
   after_script:
     - rm -rf .git/ ./*
 
+publish-subsystem-benchmarks:
+  stage: publish
+  variables:
+    CI_IMAGE: "paritytech/tools:latest"
+  rules:
```

Comment:
seems like `rules` is redundant, since job extends .publish-gh-pages-refs.

Reply:
thanks, forgot to remove them, fixed in https://github.com/paritytech/polkadot-sdk/pull/3853/commits/92c438a1b62569577be2b3d4b463bbb9cb3de0d4